core/src/test/java/org/infinispan/distribution/DistAsyncFuncTest.java

```diff
@@ -105,21 +100,21 @@ protected void asyncWait(Object key, Predicate<VisitableCommand> command) {
          for (ReplListener rl : r) rl.waitForRpc();
       } else {
          for (Cache<?, ?> c : getOwners(key)) {
-            listenerLookup.get(c).expect(command);
-            listenerLookup.get(c).waitForRpc();
+//            listenerLookup.get(address(c)).expect(command);
+//            listenerLookup.get(address(c)).waitForRpc();
```

Comment:
Oops, I was testing that the test really fails without the wait, and forgot to uncomment these afterwards.

Reply:
Yeah I actually saw that, but forgot to submit my review comments for this :)